src/index.js

```diff
@@ -61,7 +62,7 @@ export default function (CustomElement, opts) {
       });
     }
     render() {
-      return React.createElement(tagName, null, this.props.children);
+      return React.createElement(tagName, { style: this.props.style }, this.props.children);
```

Comment:
@treshugart Yup sorry
What about `{this.props}`. Can we just pass all props down without explicitly listing everything?